Source/FieldSolver/Make.package

```diff
@@ -2,9 +2,10 @@ CEXE_headers += WarpX_K.H
 CEXE_headers += WarpX_FDTD.H
 CEXE_sources += WarpXPushFieldsEM.cpp
 ifeq ($(USE_PSATD),TRUE)
-  CEXE_sources += WarpXFFT.cpp
-  F90EXE_sources += WarpX_fft.F90
```

Comment:
These files have been moved into the folder `PicsarHybridSpectralSolver`, and renamed.